packages/svelte/src/internal/client/render.js

```diff
@@ -83,6 +83,13 @@ export function template(html, is_fragment) {
 		if (cached_content === undefined) {
 			const content = create_fragment_from_html(html);
 			cached_content = is_fragment ? content : /** @type {Node} */ (child(content));
+			if (DEV && cached_content instanceof Element && cached_content.outerHTML !== html) {
```

Comment:
have to be careful here, browsers might do some normalization on the order of attrs, style rules etc. which may lead to false positives in some cases although I don't remember exactly which ones, I don't think our test suite is set up to catch this

+ in the future we might want to leverage some html parsing tricks for smaller codegen, but i guess for that it can just generate different templates in dev vs. prod

Reply:
Yeah stumbled into that one already. Think this PR is dead because of that. There's no other way I can find. I tried to apply some wrapping element in another namespace to avoid the browser autofix but nothing works.